.changeset/nice-ligers-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@web/dev-server-storybook': patch
```

Comment:
While _technically_ these are breaking changes, I'm fine with this since the current major version never worked and these changes should have been included in that breaking change. This is just fixing the broken `1.0` version.

Reply:
The API is the same but I'm deprecating the CommonJS built code that's used when the package is "required".

Reply:
That is what the 1.0 intended to do, right? In which case, I’m even more down with this as a patch.